src/consts.ts

```diff
@@ -0,0 +1,3 @@
+const API_URL = "example.com";
```

Comment:
I see the improvement here (to read these props from the `.env` file instead of hardcoding this here).